pandas/tests/arrays/test_datetimelike.py

```diff
@@ -564,7 +564,7 @@ def test_shift_fill_int_deprecated(self):
 
         expected = arr.copy()
         if self.array_cls is PeriodArray:
-            fill_val = PeriodArray._scalar_type._from_ordinal(1, freq=arr.freq)
+            fill_val = arr._scalar_type._from_ordinal(1, freq=arr.freq)
```

Comment:
This was causing the error: when accessing a property on the class itself, you get the property object, not its content.